<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralForm</class>
 <widget class="QDialog" name="GeneralForm">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>549</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>549</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Agregar Ficha</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="cuentaLabel">
       <property name="minimumSize">
        <size>
         <width>136</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>136</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Cuenta Corriente:</string>
       </property>
       <property name="buddy">
        <cstring>cuentaCorrienteEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cuentaCorrienteEdit">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="inputMask">
        <string>00-0000-00;_</string>
       </property>
       <property name="text">
        <string>--</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="numeroLabel">
       <property name="minimumSize">
        <size>
         <width>136</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>136</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Número de Inventario:</string>
       </property>
       <property name="buddy">
        <cstring>numeroEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="numeroEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="inputMask">
        <string notr="true">00-0000;_</string>
       </property>
       <property name="text">
        <string notr="true">-</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="direccionLabel">
       <property name="minimumSize">
        <size>
         <width>136</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>136</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Dirección:</string>
       </property>
       <property name="buddy">
        <cstring>direccionEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="direccionEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>25</height>
        </size>
       </property>
       <property name="maxLength">
        <number>150</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="numeroEdifLabel">
       <property name="minimumSize">
        <size>
         <width>136</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>136</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Número de Edificación:</string>
       </property>
       <property name="buddy">
        <cstring>numeroEdifEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="numeroEdifEdit">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>25</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="zonaLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Zona</string>
       </property>
       <property name="buddy">
        <cstring>zonaEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="manzanaLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Manzana</string>
       </property>
       <property name="buddy">
        <cstring>manzanaEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="loteLabel">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Lote</string>
       </property>
       <property name="buddy">
        <cstring>loteEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLineEdit" name="zonaEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="maxLength">
        <number>2</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="manzanaEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="maxLength">
        <number>4</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="loteEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>25</height>
        </size>
       </property>
       <property name="maxLength">
        <number>2</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="denomLabel">
       <property name="minimumSize">
        <size>
         <width>136</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>136</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Denominación:</string>
       </property>
       <property name="buddy">
        <cstring>denomEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="denomEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>25</height>
        </size>
       </property>
       <property name="maxLength">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="denomHistLabel">
       <property name="text">
        <string>Denominación Historica:</string>
       </property>
       <property name="buddy">
        <cstring>denomHistEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="denomHistEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>25</height>
        </size>
       </property>
       <property name="maxLength">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="barrioLabel">
       <property name="minimumSize">
        <size>
         <width>136</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>136</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Barrio:</string>
       </property>
       <property name="buddy">
        <cstring>barrioIdEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="barrioIdEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="barrioEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="encuestadorLabel">
       <property name="minimumSize">
        <size>
         <width>136</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>136</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Encuestador:</string>
       </property>
       <property name="buddy">
        <cstring>encIdEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="encIdEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>25</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="encEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>25</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="agregarEncButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="centrohistorico.qrc">
         <normaloff>:/icons/encuestadores.png</normaloff>:/icons/encuestadores.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="9" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="fechaLabel">
       <property name="minimumSize">
        <size>
         <width>136</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>136</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Fecha:</string>
       </property>
       <property name="buddy">
        <cstring>fechaEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="fechaEdit">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="displayFormat">
        <string>dd/MM/yyyy</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="cataLabel">
       <property name="minimumSize">
        <size>
         <width>136</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>136</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Catalogación:</string>
       </property>
       <property name="buddy">
        <cstring>cataEdit</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cataEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>25</height>
        </size>
       </property>
       <property name="maxLength">
        <number>30</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="11" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QLabel" name="comentLabel">
       <property name="minimumSize">
        <size>
         <width>137</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>150</height>
        </size>
       </property>
       <property name="text">
        <string>Comentarios:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="commentEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="12" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>150</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Guardar</string>
       </property>
       <property name="icon">
        <iconset resource="centrohistorico.qrc">
         <normaloff>:/icons/guardar.png</normaloff>:/icons/guardar.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancelar</string>
       </property>
       <property name="icon">
        <iconset resource="centrohistorico.qrc">
         <normaloff>:/icons/close.png</normaloff>:/icons/close.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>150</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="centrohistorico.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>barrioEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>GeneralForm</receiver>
   <slot>onBarrioTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>encEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>GeneralForm</receiver>
   <slot>onEncuestadorTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>332</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>GeneralForm</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>298</x>
     <y>409</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zonaEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>GeneralForm</receiver>
   <slot>updateCuentaCorriente()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>49</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>manzanaEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>GeneralForm</receiver>
   <slot>updateCuentaCorriente()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loteEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>GeneralForm</receiver>
   <slot>updateCuentaCorriente()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>GeneralForm</receiver>
   <slot>guardar()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>409</y>
    </hint>
    <hint type="destinationlabel">
     <x>257</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cuentaCorrienteEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>GeneralForm</receiver>
   <slot>onCuentaCorrienteTextChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>agregarEncButton</sender>
   <signal>clicked()</signal>
   <receiver>GeneralForm</receiver>
   <slot>abrirNuevoEncuestador()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>486</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>266</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onBarrioTextChanged(QString)</slot>
  <slot>onEncuestadorTextChanged(QString)</slot>
  <slot>updateCuentaCorriente()</slot>
  <slot>guardar()</slot>
  <slot>onCuentaCorrienteTextChanged(QString)</slot>
  <slot>abrirNuevoEncuestador()</slot>
 </slots>
</ui>
